src/ralph/defaults.py

```diff
@@ -35,9 +38,72 @@ class StorageBackends(Enum):
     FS = "ralph.backends.storage.fs.FSStorage"
 
 
+def load_config():
+    """Return a dictionnary representing Ralph's configuration"""
+
+    config_path = APP_DIR / ".config.yml"
+
+    if not config_path.is_file():
+        return None
+
+    try:
+        with open(config_path, "r") as config_file:
+            config = yaml.safe_load(config_file)
+            return config
+    except Exception as exc:
+        raise ConfigurationException("Configuration file is not valid") from exc
+
+
+def get_value(key, default_value):
+    """
+    Get a value based on its key returning the first of (in order):
+        1. Environment
+        2. Config file
+        3. default_value
+    """
+
+    config = load_config()
```

Comment:
IMO you should load it once in a `CONFIG` variable

Reply:
It is possible and we fixed it this way, however it generates many linting errors, since it needs a cache and we have to ignore flake8 errors